spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -213,21 +190,19 @@
       expect(authorship_matches.length).to eq(1)
       authorship_matches.first
     end
-
-    it_behaves_like 'it is successful'
+    let(:request_data) { update_authorship_for_pub_with_contributions }
 
     it 'does not create a new contribution' do
       expect { http_request }.not_to change(Contribution, :count)
+      expect(response).to have_http_status(:success)
     end
 
     it 'updates all contribution attributes' do
       http_request # defined in context, uses request_data
       existing_contrib.reload
-      expect(request_data[:featured]).not_to be_nil
+      expect(request_data).to include(featured: !be_nil, status: be_present, visibility: be_present)
       expect(existing_contrib.featured).to be request_data[:featured]
-      expect(request_data[:status]).not_to be_nil
       expect(existing_contrib.status).to eq request_data[:status]
-      expect(request_data[:visibility]).not_to be_nil
       expect(existing_contrib.visibility).to eq request_data[:visibility]
     end
```

Comment:
`!be_nil` is different from `be_present`?  If this is in shared_examples, it might be shared between various HTTP methods, hence the check on the request payload(s); i.e. a PATCH could omit some.

Reply:
Yeah, `!be_nil` is required for the boolean, because false is not "present" but it is not nil.

Reply:
Ah, right, yea.